gistBFO.ttl

```diff
@@ -194,26 +180,44 @@ gist:OrderedMember
 	.
 
 gist:Organization
-	rdfs:subClassOf obo:BFO_0000141 ;
+	rdfs:subClassOf obo:BFO_0000027 ;
 	gist:bfoMappingNote "Organization is clearly an independent continuant and immaterial. May need to update to be consistent with Common Core Ontology as well."^^xsd:string ;
 	.
 
 gist:PhysicalIdentifiableItem
 	rdfs:subClassOf obo:BFO_0000004 ;
-	skos:bfoMappingNote "A physical identifiable item is a independent continuant that is not a spatial region."^^xsd:string ;
+	gist:bfoMappingNote "A physical identifiable item is a independent continuant that is not a spatial region."^^xsd:string ;
 	.
 
 gist:PhysicalSubstance
 	rdfs:subClassOf obo:BFO_0000040 ;
 	gist:bfoMappingNote "Physical substance is a material entity, but not an object or aggregate as those are physical identifiable items and collections."^^xsd:string ;
 	.
 
+gist:Place
+	rdfs:subClassOf [
+		a owl:Class ;
+		owl:unionOf (
+			obo:BFO_0000029
+			obo:BFO_0000040
+			obo:BFO_0000140
+		) ;
+	] ;
+	.
+
 gist:SchemaMetaData
 	rdfs:subClassOf obo:BFO_0000031 ;
 	.
 
 gist:Specification
-	rdfs:subClassOf obo:BFO_0000031 ;
+	rdfs:subClassOf [
+		a owl:Class ;
+		owl:unionOf (
+			obo:BFO_0000020
+			obo:BFO_0000031
+		) ;
+	] ;
+	gist:bfoMappingNote "Likely a CCO directive information content entity"^^xsd:string ;
```

Comment:
@giacomodecolle I'm thinking `Specification` is definitely a GDC (and probably, as you say, a directive ICE). But the superclass of `Intention` could be `GDC or SDC`.